src/App.jsx

```diff
@@ -0,0 +1,29 @@
+import { Route, Routes } from "react-router-dom";
+import HelicopterList from "./components/Helicopter";
+import Details from "./pages/Details";
+import AddItem from "./components/AddItem";
+import Login from "./components/sessions/login";
+import Register from "./components/sessions/signup";
+import Reserve from "./components/reserve";
+import Reservations from "./components/reservations";
+import DeleteReservationComponent from "./components/DeleteReservation";
+import DeleteHelicopterComponent from "./components/DeleteHelicopter";
+
+
+const App = () => {
```

Comment:
- [ ] In the details page, the "reserve" button does not work:

![image](https://github.com/emmanuelsteven/heli-hire-captsone-frontend/assets/57421823/e28860ed-131d-441d-970b-944c5e6b6185)

Kindly, make it so it takes your users to the reservations page. 👍

Reply:
fixed